packages/rest/src/rest.application.ts

```diff
@@ -87,7 +87,6 @@ export class RestApplication extends Application implements HttpServerLike {
    * See https://expressjs.com/en/4x/api.html#express.static
    * @param path The path(s) to serve the asset.
    * See examples at https://expressjs.com/en/4x/api.html#path-examples
-   * To avoid performance penalty, `/` is not allowed for now.
```

Comment:
The changes in this file look very unrelated to the rest of this pull request. Could you please move them to a new PR? Or at least to a standalone commit.